app/controllers/host_aggregate_controller.rb

```diff
@@ -0,0 +1,156 @@
+class HostAggregateController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def index
+    redirect_to :action => 'show_list'
+  end
```

Comment:
I am seeing this in a lot of newly added controllers, but I am quite sure it's not needed in many cases. In this PR you also add the route for this, but is it really going to be used? Do we need both `index` and `show_list`?